src/test/scala/mesosphere/marathon/tasks/InstanceTrackerImplTest.scala

```diff
@@ -219,30 +222,12 @@ class InstanceTrackerImplTest extends AkkaUnitTest {
       val appName2 = "app2".toRootPath
       val appName3 = "app3".toRootPath
 
-      val app1_instance1 = makeSampleInstance(appName1)
-      val app1_instance2 = makeSampleInstance(appName1)
-      val app2_instance1 = makeSampleInstance(appName2)
-      val app3_instance1 = makeSampleInstance(appName3)
-      val app3_instance2 = makeSampleInstance(appName3)
-      val app3_instance3 = makeSampleInstance(appName3)
-
-      instanceTracker.launchEphemeral(app1_instance1).futureValue
-      instanceTracker.updateStatus(app1_instance1, makeTaskStatus(app1_instance1, TaskState.TASK_RUNNING), clock.now()).futureValue
-
-      instanceTracker.launchEphemeral(app1_instance2).futureValue
-      instanceTracker.updateStatus(app1_instance2, makeTaskStatus(app1_instance2, TaskState.TASK_RUNNING), clock.now()).futureValue
-
-      instanceTracker.launchEphemeral(app2_instance1).futureValue
-      instanceTracker.updateStatus(app2_instance1, makeTaskStatus(app2_instance1, TaskState.TASK_RUNNING), clock.now()).futureValue
-
-      instanceTracker.launchEphemeral(app3_instance1).futureValue
-      instanceTracker.updateStatus(app3_instance1, makeTaskStatus(app3_instance1, TaskState.TASK_RUNNING), clock.now()).futureValue
-
-      instanceTracker.launchEphemeral(app3_instance2).futureValue
-      instanceTracker.updateStatus(app3_instance2, makeTaskStatus(app3_instance2, TaskState.TASK_RUNNING), clock.now()).futureValue
-
-      instanceTracker.launchEphemeral(app3_instance3).futureValue
-      instanceTracker.updateStatus(app3_instance3, makeTaskStatus(app3_instance3, TaskState.TASK_RUNNING), clock.now()).futureValue
+      val app1_instance1 = TestInstanceBuilder.runningInstance(appName1, Timestamp.now(), instanceTracker).futureValue
+      val app1_instance2 = TestInstanceBuilder.runningInstance(appName1, Timestamp.now(), instanceTracker).futureValue
+      val app2_instance1 = TestInstanceBuilder.runningInstance(appName2, Timestamp.now(), instanceTracker).futureValue
+      val app3_instance1 = TestInstanceBuilder.runningInstance(appName3, Timestamp.now(), instanceTracker).futureValue
+      val app3_instance2 = TestInstanceBuilder.runningInstance(appName3, Timestamp.now(), instanceTracker).futureValue
+      val app3_instance3 = TestInstanceBuilder.runningInstance(appName3, Timestamp.now(), instanceTracker).futureValue
```

Comment:
What is the motivation for using a helper instead of calling the instance tracker directly? This is a test for the instance tracker. It should be explicit that we call it.